ui-tests/src/test/resources/features/integrations/split-aggregate-steps.feature

```diff
@@ -0,0 +1,87 @@
+# @sustainer: mastepan@redhat.com
+
+@ui
+@integrations-db-to-db
```

Comment:
+ maybe add `@split` `@aggregate`, @mcada wdyt

Reply:
@split-aggregate
I think this steps should always work together

Reply:
afaik you can also have `Start -> Split -> End` without aggregate step.

Reply:
The idea behind scenario tags is that you have an easy option to run all the tests where `@split` or `@aggregate` options are used. If you can use just split step without the other one, it would be better to have two tags as it changes nothing but adds an option to run just one of them. I personally have never used these "group" tags, but one never knows, maybe there will be a one-off for `@split` ^^